README.md

```diff
@@ -1,55 +1,37 @@
 # iRODS TypeScript HTTP API Wrapper
 
-## Installation and Usage
-
-1. Install npm package: 
+## Installation
 
 ```
 npm install git+https://github.com/irods/irods_client_http_typescript
 ```
 
-2. Import `IrodsHttpClient` class:
+## Usage
+
+After creating an instance of the wrapper, you can authenticate it within an asynchronous context, then do any operations available from the http api. The following code demonstrates a way of creating a new collection with the wrapper.
```

Comment:
This statement can be reduced to the following:
```
The following example demonstrates how the client can be used to create a new collection.
```

Reply:
Done.